simulation_ws/src/cloudwatch_simulation/launch/turtlebot3_navigation.launch.py

```diff
@@ -7,6 +7,9 @@
 
 
 def generate_launch_description():
+    TURTLEBOT3_MODEL = os.environ['TURTLEBOT3_MODEL']
```

Comment:
Can get rid of this whole file now that you have turtlebot3_nav2. Or rename turtlebot3_nav2 to this.